ansible/playbooks/tasks/cluster-hana.yaml

```diff
@@ -89,19 +151,27 @@
   changed_when: false
 
 - name: Ensure maintainence mode is disabled
-  ansible.builtin.command: crm configure property maintenance-mode=false
+  ansible.builtin.command:
+    cmd: crm configure property maintenance-mode=false
   when:
     - is_primary
     - refreshed_crm_maintainence_mode is true or refreshed_crm_maintainence_mode == 'unknown'
 
 - name: Get cluster state
-  ansible.builtin.command: crm status
+  ansible.builtin.command:
+    cmd: crm status
   register: reg_crm_status
   when: is_primary
 
 - name: Cleanup if needed
-  # TODO this task has hardcoded SID and instance number values which should be variables!
-  ansible.builtin.command: "crm resource cleanup {{ rsc_SAPHana }}"
+  ansible.builtin.command:
+    cmd: "crm resource cleanup {{ rsc_SAPHana }}"
   when:
     - reg_crm_status.stdout | regex_search('Failed Resource Actions') | trim | length != 0
     - is_primary
+
+# For debug purpose only
```

Comment:
New code useful to "see" a summary of the configuration at the end

Reply:
Awesome! Does the regex means that it's only printed when there are errors?